libs/wingsdk/src/util/util.ts

```diff
@@ -57,6 +208,75 @@ export interface NanoidOptions {
  * Utility functions.
  */
 export class Util {
+  /**
+   * Execute a command in the shell and return its standard output.
+   * @param command - The shell command to execute.
+   * @param opts - The shell options including working directory, environment variables, etc.
+   * @returns The standard output of the shell command.
+   * @throws An error if the command returns a non-zero exit code.
+   */
+  public static async shell(
+    command: string,
+    opts?: ShellOptions
+  ): Promise<string> {
+    const args = command.split(" ");
+    const output = await Util.exec("/bin/sh", ["-c", ...args], opts);
+    if (output.status !== 0) {
+      throw new Error(
+        `Command exited with status ${output.status}: ${command}`
+      );
+    }
+    return output.stdout;
+  }
+
+  /**
+   * Execute a program with the given arguments, wait for it to finish, and
+   * return its outputs.
+   * @param program - The program to execute.
+   * @param args - An array of arguments to pass to the program.
+   * @param opts - Execution options including working directory, environment variables, etc.
+   * @returns A promise that resolves to the output of the executed program.
+   * @throws An error if the execution fails.
+   */
+  public static async exec(
+    program: string,
+    args: Array<string>,
+    opts?: ExecOptions
+  ): Promise<Output> {
+    const command = `${program} ${args.join(" ")}`;
+
+    try {
+      const { stdout, stderr } = await execPromise(command, opts);
```

Comment:
There's a small issue - Wing's `shell()` function is intended to run the user's command in a shell, but the `exec()` function should not. To run the program directly, I think we need to use [execFile](https://nodejs.org/api/child_process.html#child_processexecfilefile-args-options-callback) instead of exec. Can you try this and see if it's possible to get the tests passing?

execFile will allow you to pass the program and args directly (no need to concatenate the strings together)

Reply:
Essentially, the current `exec.test.w` should be renamed to `shell.test.w`. As for `execFile()`, what kind of program could we run with it in a test?